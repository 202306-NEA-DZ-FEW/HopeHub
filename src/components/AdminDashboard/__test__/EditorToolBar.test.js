import renderer from "react-test-renderer";

import EditorToolBar from "../EditorToolBar";

it("renders correctly", () => {
    const tree = renderer.create(<EditorToolBar />).toJSON();
    expect(tree).toMatchSnapshot();
});
