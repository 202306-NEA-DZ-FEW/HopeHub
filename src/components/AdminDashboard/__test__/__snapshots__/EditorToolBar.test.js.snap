// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  id="toolbar"
>
  <span
    className="ql-formats"
  >
    <select
      className="ql-font"
      defaultValue="arial"
    >
      <option
        value="arial"
      >
        Arial
      </option>
      <option
        value="comic-sans"
      >
        Comic Sans
      </option>
      <option
        value="courier-new"
      >
        Courier New
      </option>
      <option
        value="georgia"
      >
        Georgia
      </option>
      <option
        value="helvetica"
      >
        Helvetica
      </option>
      <option
        value="lucida"
      >
        Lucida
      </option>
      <option
        value="poppins"
      >
        Poppins
      </option>
    </select>
    <select
      className="ql-size"
      defaultValue="medium"
    >
      <option
        value="extra-small"
      >
        Size 1
      </option>
      <option
        value="small"
      >
        Size 2
      </option>
      <option
        value="medium"
      >
        Size 3
      </option>
      <option
        value="large"
      >
        Size 4
      </option>
    </select>
    <select
      className="ql-header"
      defaultValue="3"
    >
      <option
        value="1"
      >
        Heading
      </option>
      <option
        value="2"
      >
        Subheading
      </option>
      <option
        value="3"
      >
        Normal
      </option>
    </select>
  </span>
  <span
    className="ql-formats"
  >
    <button
      className="ql-bold"
    />
    <button
      className="ql-italic"
    />
    <button
      className="ql-underline"
    />
    <button
      className="ql-strike"
    />
  </span>
  <span
    className="ql-formats"
  >
    <button
      className="ql-list"
      value="ordered"
    />
    <button
      className="ql-list"
      value="bullet"
    />
    <button
      className="ql-indent"
      value="-1"
    />
    <button
      className="ql-indent"
      value="+1"
    />
  </span>
  <span
    className="ql-formats"
  >
    <button
      className="ql-script"
      value="super"
    />
    <button
      className="ql-script"
      value="sub"
    />
    <button
      className="ql-blockquote"
    />
    <button
      className="ql-direction"
    />
  </span>
  <span
    className="ql-formats"
  >
    <select
      className="ql-align"
    />
    <select
      className="ql-color"
    />
    <select
      className="ql-background"
    />
  </span>
  <span
    className="ql-formats"
  >
    <button
      className="ql-link"
    />
    <button
      className="ql-image"
    />
    <button
      className="ql-video"
    />
  </span>
  <span
    className="ql-formats"
  >
    <button
      className="ql-undo"
    >
      <svg
        viewBox="0 0 18 18"
      >
        <polygon
          className="ql-fill ql-stroke"
          points="6 10 4 12 2 10 6 10"
        />
        <path
          className="ql-stroke"
          d="M8.09,13.91A4.6,4.6,0,0,0,9,14,5,5,0,1,0,4,9"
        />
      </svg>
    </button>
    <button
      className="ql-redo"
    >
      <svg
        viewBox="0 0 18 18"
      >
        <polygon
          className="ql-fill ql-stroke"
          points="12 10 14 12 16 10 12 10"
        />
        <path
          className="ql-stroke"
          d="M9.91,13.91A4.6,4.6,0,0,1,9,14a5,5,0,1,1,5-5"
        />
      </svg>
    </button>
  </span>
</div>
`;
